Xamarin.Forms.Platform.iOS/Renderers/NavigationRenderer.cs

```diff
@@ -475,10 +475,24 @@ void UpdateHideNavigationBarSeparator()
 			if (_defaultNavBarShadowImage == null)
 				_defaultNavBarShadowImage = NavigationBar.ShadowImage;
 
-			if (shouldHide)
-				NavigationBar.ShadowImage = new UIImage();
+#if __XCODE11__
+			if (Forms.IsiOS13OrNewer)
+			{
+				if (shouldHide)
```

Comment:
IF we want to return to the original (with shadow) i don't it will work on iOS13

Reply:
@rmarinho  I have added missing ELSE case to restore default separator color

![separator2](https://user-images.githubusercontent.com/17849938/76794502-d7861000-67cf-11ea-81db-2172265a0c59.gif)